stories/topics/proc-aap-gcp-retrieve-admin-password.adoc

```diff
@@ -0,0 +1,16 @@
+[id="proc-aap-gcp-retrieve-admin-password"]
+
+= Retrieving the administration password
+
+Use the following procedure to retrieve the administration password.
+
+.Procedure
+. In the AWS UI, select the main menu from the top left.
```

Comment:
AWS?

Reply:
Sorry, currently working in two seperate worlds.  I'll fix this.